rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -411,8 +411,8 @@ if [ $IS_RASPI = 1 ] || [ $IS_X86 = 1 ]; then
 fi
 
 # Install Whirlpool
-WHIRLPOOL_VERSION="0.10.5"
-WHIRLPOOL_UPGRADE_URL=https://github.com/Samourai-Wallet/whirlpool-client-cli/releases/download/$WHIRLPOOL_VERSION/whirlpool-client-cli-$WHIRLPOOL_VERSION-run.jar
+# update the entire URL with every available upgrade
```

Comment:
I was about to do this update as well, but yours has more updates so I will merge it. However, can you add separate variables separate for the version and the hash and use them in the URL?

I am planning on updating the tracker files to contain the version, so it can be easily displayed somewhere in the future. Also, I might starting making some add upgrades optional and not automatic on each myNode release.

Reply:
I see. I appreciate that plan. I can make those minor changes later tonight.

Reply:
done